learning/src/components/ExampleSentences.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
```

Comment:
Also another thing is to make sure that you add semicolons at the end of all javascript expressions. I use eslint to catch styling errors like this (as well as the indentions). An eslintrc has already been set up with the conventions that we are using. All you need to do is to install the extension in your IDE, or run the command if you prefer that.